app/server.go

```diff
@@ -442,7 +440,7 @@ func (s *Server) Shutdown() error {
 		}
 	}
 
-	err := s.shutdownDiagnostics()
+	err := s.shutdownRudder()
```

Comment:
i'm 1/4 on this, but I think we should lave the names as `shutdownDiagnostics` and `initDiagnostics`. Easier for people to read the code (not everyone knows what Rudder is)